src/types/type.ts

```diff
@@ -0,0 +1,38 @@
+export interface Articles {

```

Comment:
I saw many times interfaces start from "I" in projects.
For instance: IArticles, ISourceItems